src/main/java/com/gildedrose/Conjured.java

```diff
@@ -0,0 +1,27 @@
+package com.gildedrose;
+
+public class Conjured extends DefaultItem {
+    private int conjuredQualityDegradation;
+
+    public Conjured() {
+        this.conjuredQualityDegradation = super.qualityDegradation * 2;
+    }
+
+    public void age(Item item) {
+        item.quality = updateQuality(item.quality, item.sellIn);
+        item.sellIn = updateSellIn(item.sellIn);
+    }
+
+    private int updateQuality(int quality, int sellIn) {
+        if (sellIn <= 0) {
+            conjuredQualityDegradation = super.qualityDegradation * 2;
```

Comment:
Question: Does this `conjuredQualityDegredation` value need to be set here if it's already set in the constructor on line 7?

Reply:
This actually exposed a missing edge case in our test harness that Conjured doubles after the sellIn date so if default goes to 2 when passed sellIn then Conjured should be 4. Plus the syntax needed to be `... = conjuredQualityDegredation * 2` This was addressed just now in this commit https://github.com/jramnani/GildedRoseJavaLLC/pull/16/commits/b88740b67b9b2c72b7d494e7e2ece559cfef73bd